pkg/maps/lbmap/lbmap.go

```diff
@@ -172,13 +217,96 @@ func LookupRevNat(key RevNatKey) (RevNatValue, error) {
 	return revnat.Convert(), nil
 }
 
+// gcd computes the gcd of two numbers.
+func gcd(x, y uint16) uint16 {
+	for y != 0 {
+		x, y = y, x%y
+	}
+	return x
+}
+
+// Generates a wrr sequence based on provided weights.
```

Comment:
Is this function used outside this package? If not it can be set to private `GenerateWrrSeq` - > `generateWrrSeq`?